sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/Models/TelemetryEventData.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using OpenTelemetry.Logs;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Models
+{
+    internal partial class TelemetryEventData
+    {
+        public TelemetryEventData(int version, LogRecord logRecord) : base(version)
+        {
+            Properties = new ChangeTrackingDictionary<string, string>();
+            Measurements = new ChangeTrackingDictionary<string, double>();
+
+            Name = logRecord.EventId.Name;
+
+            if (logRecord.Attributes != null)
+            {
+                foreach (var kv in logRecord.Attributes)
+                {
+                    Properties.Add(kv.Key, kv.Value?.ToString());
```

Comment:
if we are stringifying here anyway, could the API just accept kvp of string,string, instead of string,object?

Reply:
Sdk wouldn't be able to detect the state in that case: https://github.com/open-telemetry/opentelemetry-dotnet/blob/1b3f1894d95fd01fe475245ec60ef273d5f77bb6/src/OpenTelemetry/Logs/ILogger/OpenTelemetryLogger.cs#L166-L175